.github/dependabot.yml

```diff
@@ -1097,8 +1102,3 @@ updates:
     schedule:
       interval: "weekly"
       day: "wednesday"
-  - package-ecosystem: "gomod"
-    directory: "/testbed"
```

Comment:
This seems incorrect

Reply:
yeah.. it is because of the limit of 220 entries in dependabot... each time a new component is added to dependabot, the last one is removed from the list

-  /testbed/mockdatareceivers/mockawsxrayreceiver was cut here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/19379
- /testbed/mockdatasenders/mockdatadogagentexporter was cut here https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/19377